SC_redesign/Crop_and_Soil/crop/crop_data.py

```diff
@@ -51,6 +53,24 @@ class CropData:
     is_alive: bool = True
     """is the crop currently alive in the field?"""
 
+    # Management variables
+    planting_year: int = 0
+    """the year that this crop was planted"""
+    planting_day: int = 100
+    """the (julian) day that this crop was planted"""
+    next_harvest_year: int = 0
+    """the year in which this crop should be harvested next"""
+    next_harvest_day: int = 250
+    """the (julian) day on which this crop should be harvested next"""
+    use_heat_scheduling: bool = False
+    """should heat unit scheduling be used to determine harvesting instead of user-specified harvest dates?"""
```

Comment:
Not sure about the formality of the documentation. I think that putting a question as attribute documentation is kinda odd. I'm not that fluent in English but perhaps something like this?
```suggestion
    """Status indicating if heat unit scheduling will be used to determine harvesting instead of user-specified harvest dates"""
```

Reply:
I like this pattern for boolean attributes.

Reply:
@Isgotkowitz, to clarify, do you like the original wording or Matthew's alterations?

Reply:
Matthew's alterations

Reply:
OK good. I agree, just checking. Thanks guys.